cpp/src/backends/torch_scripted/torch_scripted_backend.cc

```diff
@@ -74,14 +74,31 @@ std::unique_ptr<torchserve::LoadModelResponse> Backend::LoadModelInternal(
 
 std::shared_ptr<torchserve::InferenceResponseBatch> ModelInstance::Predict(
     std::shared_ptr<torchserve::InferenceRequestBatch> request_batch) {
+  auto start_time = std::chrono::high_resolution_clock::now();
+
   auto response_batch = std::make_shared<torchserve::InferenceResponseBatch>();
   for (const auto& request : (*request_batch)) {
     (*response_batch)[request.request_id] =
         std::make_shared<torchserve::InferenceResponse>(request.request_id);
   }
   handler_->Handle(model_, device_, request_batch, response_batch);
 
+  auto stop_time = std::chrono::high_resolution_clock::now();
+  std::chrono::duration<double, std::milli> duration = stop_time - start_time;
+  try {
+    auto& prediction_time_metric =
+        torchserve::MetricsRegistry::GetMetricsCacheInstance()->GetMetric(
+            torchserve::MetricType::GAUGE, "PredictionTime");
+    prediction_time_metric.AddOrUpdate(
+        std::vector<std::string>{manifest_->GetModel().model_name, "Model"},
+        duration.count());
+  } catch (std::runtime_error& e) {
+    TS_LOG(ERROR, e.what());
+  } catch (std::invalid_argument& e) {
+    TS_LOGF(ERROR, "Failed to record metric. {}", e.what());
+  }
+
```

Comment:
the request_id is not included in the log metrics. it cause different format model_metrics.log b/w python and cpp. it may break cx parser pipeline.